src/connector.js

```diff
@@ -356,47 +367,19 @@ function initTpl() {
 	if ( !suggestionsElement && searchBoxElement && params.unsupportedSuggestions && params.numberOfSuggestions > 0 ) {
```

Comment:
Please remove `params.unsupportedSuggestions` as part of the condition.
```suggestion
	if ( !suggestionsElement && searchBoxElement && params.numberOfSuggestions > 0 ) {
```

Reply:
Done in latest commit